docs/UserGuide.md

```diff
@@ -170,81 +170,125 @@ Examples:
 
-Format: `find FIELD/KEYWORD [FIELD/KEYWORD]`
+If you then find that the basic filtering is insufficient for your use case, you can read the [advanced filtering](#find-contacts-advanced-filtering) section to learn how to perform more complex filtering.
 
-You may include multiple field-keyword combinations. The `find` command will perform a logical `AND` on all field-keyword combinations.
 
-#### Search contact by name
 
-Locates persons in the address book whose names contain the given keywords.
+<panel header="**Supported Fields**" type="primary" id="find-fields-table" expanded no-close>
+Across both basic and advanced filtering, the following fields are supported:<br><br>
 
-Format: `find name/KEYWORD`
+| Field | Prefix | Description |
+| ----- | ------ | ----------- |
+| Name  | `n`   | Finds contacts whose names **contain** the given keyword. |
+| Phone | `p`   | Finds contacts whose phone numbers **contain** the given digits. |
+| Email | `e`   | Finds contacts whose email addresses **contain** the given keyword. |
+| Address | `a` | Finds contacts whose addresses **contain** the given keyword. |
+| Tag   | `t`   | Finds contacts who have any tag that **exactly matches** the given keyword. |
 
-Examples:
-* `find name/John`
-  * Returns all contacts with names that contain "John" (e.g. "John Smith", "John Doe")
-* `find name/Alex name/Ye`
-  * Returns all contacts with names that contain both "Alex" and "Ye" (e.g. "Alex Yeoh", "Yervis Alexis")
+</panel>
 
-#### Search contact by phone number
+<br>
 
-Locates persons in the address book whose phone numbers contain the given digits.
+<box type="info">
 
-Format: `find phone/KEYWORD`
+Note that in all cases, the search is case-insensitive for alphabetic characters. For example, `n/Joe` will match contacts who have the name `Joe`, `joE`, `Ajoeia`, `BobJOe Lee`, etc., and `t/friend` will match `friend`, `FriENd`, `FRIEND`, etc.
+</box>
 
-Examples:
-* `find phone/9876`
-  * Returns all contacts with phone numbers that contain "9876"
 
-#### Search contact by email
+<box type="warning">
 
-Locates persons in the address book whose email addresses contain the given keywords.
+For now, search keywords cannot contain spaces. For example, `n/John Doe` will not work as expected. Functionality to search for keywords which spaces like `"John Doe"` will be added in a future release.
+</box>
 
-Format: `find email/KEYWORD`
+#### Find contacts: basic filtering
 
-Examples:
-* `find email/tan email/@u.nus.edu`
-  * Returns all contacts with email addresses that contain "tan" and "@u.nus.edu"
+Contacts can be filtered by a single field by typing:
+- the **prefix** of the field you're searching through, followed by
+- a **slash** (`/`), followed by
+- the **keyword** you're looking for. Such a search will return all contacts whose field matches the keyword based on the behavior specified in the [supported fields table](#find-fields-table).
```

Comment:
perhaps consider breaking the line since the line after `the **keyword** you're looking for.` has no relation with the bullet points? Consider the following

```
- the **keyword** you're looking for.

Such a search will return all contacts whose field matches the keyword based on the behavior specified in the [supported fields table](#find-fields-table).
```